_layouts/sidebar.md

```diff
@@ -9,4 +9,5 @@
   - [Additional resources](/08-additional-resources.md)
   - [Conclusion](/09-conclusion.md)
   - [Glossary](/10-glossary.md)
+  - [Understanding merge conflicts](/11-understanding-merge-conflicts)
```

Comment:
This appeared when I clicked on the Conflicts section
![D1B913CB-5A06-4CD5-B35C-D6C3673BA99C](https://github.com/open-sauced/intro/assets/105683440/cf8b9311-f75a-450f-a8f4-f6ccb7e4542e)